debugtools/DDR_Artifacts/specs/29/aix_ppc

```diff
@@ -19446,6 +19446,7 @@ C|RootScannerEntity_FinalizableObjects
 C|RootScannerEntity_JNIGlobalReferences
 C|RootScannerEntity_JNIWeakGlobalReferences
 C|RootScannerEntity_JVMTIObjectTagTables
+C|RootScannerEntity_DoubleMappedObjects
```

Comment:
These changes shouldn't be required.  DDR should generate the right data at runtime.  @keithc-ca to confirm.

Reply:
That's right: none of the files in `debugtools/DDR_Artifacts` need be modified; in fact, I've been meaning to remove them from openj9 altogether.